packages/showcases/src/styles/globals.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+.annotationLayer {
```

Comment:
Let's reduce global styles as much as possible. `annotationLayer` seems like it should be a declared in pdf page only